src/ui/navbar-header.svelte

```diff
@@ -31,15 +33,18 @@
     display: flex;
     align-items: center;
     cursor: pointer;
+    font-size: 14px;
```

Comment:
Shall we define a text style for this?
Something like `body2`, any name works, and even if it's just for this label it's fine, I just would like to have all the text styles in the text.ts file.

Reply:
yeah we talked about this and Iga said that while she was thinking on what new variation to create it was fine for a one off, but I'll create a "larger caption or something" in text